dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/impl/UriBuilderImpl.java

```diff
@@ -857,8 +857,8 @@ private String buildParams(MultivaluedMap<String, String> map, char separator) {
                     } else {
                         val = URITemplate.createExactTemplate(val).encodeLiteralCharacters(isQuery);
                     }
+                    b.append('=');
                     if (!val.isEmpty()) {
```

Comment:
Same as above - let's get rid of these files since they are identical to the Apache source.